app.yaml

```diff
@@ -19,3 +19,7 @@ handlers:
 
 includes:
   - ./env.yaml
+
+libraries: 
+  - name: ssl
```

Comment:
Which functionality requires this `ssl` library to be added? What error text do you get if this library is not present?

Reply:
Yesterday I had an error , stating that I cant call a https url . its removed now

Reply:
Understood. Please also remove the blank lines that remain in `app.yaml`.